core-bundle/tests/EventListener/DataContainer/ContentCompositionListenerTest.php

```diff
@@ -814,7 +814,7 @@ public function testCannotPasteAfterArticleIfPageIsNotFound(): void
 
     public function testCannotPasteAfterArticleIfProviderDoesNotSupportContentComposition(): void
     {
-        $page = $this->expectPageFindByPk(17, $this->pageRecord);
+        $page = $this->expectPageFindByPk($this->pageRecord);
```

Comment:
```suggestion
        $page = $this->expectPageFindByPk();
```

`$this->pageRecord` can be omitted too I think.

Reply:
Changed in 3fa8d4e0e56b089d8476fab4976d0d13729d7f2d.